### DISCUSSION TITLE
шиммер токены где искать

### DISCUSSION THREAD

**Discussion by chechet75** [iotaledger/firefly#7583] [Feedback]
всем добра,после обновления 2,0 на кошельке иота пропали токены шиммер,как их найти?всем спасибо кто откликнется с помощью.

**Comment by begonaalvarezd** [comment]
Quick information about SMR: what the old firefly version displayed was just a representation of what you owned, but they were not the real shimmers. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

Will close the issue because the new firefly app is not intended to display old staking rewards, your shimmer tokens should be managed from the shimmer app. 

Please let us know if you have any other issues
